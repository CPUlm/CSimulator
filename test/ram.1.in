8
00
1
01
1000
01
0
00
0000
11
1
11
0101
11
1
11
0111
11
0
10
0001
10
0
10
0010
10
1
00
1001
11
1
10
1110
