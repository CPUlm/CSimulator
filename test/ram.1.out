Step 1:
=> o = 0000
Step 2:
=> o = 1000
Step 3:
=> o = 0000
Step 4:
=> o = 0101
Step 5:
=> o = 0111
Step 6:
=> o = 0000
Step 7:
=> o = 0000
Step 8:
=> o = 0111

